corehq/apps/cleanup/tests/test_utils.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+
+from corehq.apps.cleanup.utils import DeletedDomains
+from corehq.apps.domain.shortcuts import create_domain
+
+
+class TestDeletedDomains(TestCase):
+
+    def test_is_domain_deleted_returns_true_for_deleted_domain(self):
+        self.assertTrue(
+            DeletedDomains().is_domain_deleted(self.deleted_domain.name))
+
+    def test_is_domain_deleted_returns_false_for_active_domain(self):
+        self.assertFalse(
+            DeletedDomains().is_domain_deleted(self.active_domain.name))
+
+    def test_is_domain_deleted_returns_false_for_inactive_domain(self):
+        self.assertFalse(
+            DeletedDomains().is_domain_deleted(self.inactive_domain.name))
+
```

Comment:
Should there be a test here for a domain that is deleted _and_ is also "active"? I'm not sure why `is_domain_deleted` even cares about whether the domain is active or not. If it's been deleted (hard or soft), then I would expect `is_domain_deleted` to return true.

Reply:
On second thought, I could keep the DeletedDomains class around to still enable caching the call to the deleted domains view 👍🏻

Reply:
I'm assuming it is still more performant to make the call to the deleted domains view once, cache it, and check against that instead of fetching each specific domain document and checking if it is of type "Domain-Deleted".